### PR TITLE
Updates to address issue Why should payload type 0 be standardized? #9

### PR DESCRIPTION
**Pull request by IainCRobertson**

### CODE REVIEW

File: e-trace-specifics.adoc [hunk 1]
```diff
@@ -20,10 +20,13 @@ Note that some E-Trace packets may optionally include a *time* field, as an alte
 
 === *type*
 
-For E-Trace, the *type* field in the *payload* group is 2 bits long with the following encodings defined:
+The capabilities of the trace source will determine the minimum width of the *type* field in the *payload* group.  For E-Trace:
```
Comment (ved-rivos) [inline e-trace-specifics.adoc:23]: The term "payload group" is only used here. The  section 2.1.4 refers to the set of `type` and `trace_payload` fields are Payload. If the "payload group" was not intended to have a different meaning then could we remove "group" from this sentence?

Comment (IainCRobertson) [reply e-trace-specifics.adoc:23]: You're right that this is the only place it's used directly.  Indirectly, it's mentioned in Table 1 and the associated text which talks about "field-groups".  But I'm okay to remove "group" from the text here if you prefer, as I don't think it's adding any value.

Comment (ved-rivos) [reply e-trace-specifics.adoc:23]: I think its slightly preferred to remove "group" to avoid some people being puzzled.

**Review by ved-rivos — APPROVED** [review]

File: e-trace-specifics.adoc (hunk 1 above)
Comment (IainCRobertson) [reply e-trace-specifics.adoc]: I've updated to use "field-group" and published to the remote repo.

**Pull request merged**